### PR TITLE
LowerCase tags and incorrect expand on cta in home page

### LINKED ISSUE #192: lower case tags
tags are currently being capitalized: `tags: ["Lightning","C-Lightning","Lnd","Wallet","Layer 2"]`

They should be lower cased.

### LINKED ISSUE #213: Mobile CTA incorrectly expands lengthwise when interacted with and doesn't scroll to anchor
Replicated on two Android phones - one using Chrome, one using Brave

Clicking "Tell me more" makes the CTA card longer and doesn't scroll down the relevant section as it should. 

The slug becomes /#need-section

image

Reopening the mobile site with the full link (https://staging-review.btctranscripts.com/#need-section) does successfully bring you to the correct anchor

### PR DESCRIPTION
**Pull request by 0tuedon**
This PR closes #213 and closes #192

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -139,7 +139,9 @@ export function getRequestUrl(req: NextApiRequest) {
 }
 
 export function formatDataForMetadata(data: string[] | string) {
+  console.log(data)
```
Comment (Extheoisah) [inline src/utils/index.ts:142]:
```suggestion
```

File: src/utils/index.ts [hunk 2]
```diff
@@ -139,7 +139,9 @@ export function getRequestUrl(req: NextApiRequest) {
...
   if (Array.isArray(data)) {
+
```
Comment (Extheoisah) [inline src/utils/index.ts:144]:
```suggestion
```

File: src/components/sideBarContentEdit/SelectField.tsx [hunk 3]
```diff
@@ -222,7 +222,9 @@ export const OnlySelectField = ({
             gap={1}
             alignItems="center"
           >
-            <Text fontSize="14px">{speaker}</Text>
+            <Text textTransform={"capitalize"} fontSize="14px">
```
Comment (Extheoisah) [inline src/components/sideBarContentEdit/SelectField.tsx:225]: Isn't this supposed to be lowercase instead of capitalize?

Comment (0tuedon) [reply src/components/sideBarContentEdit/SelectField.tsx:225]: @Emmanuel-Develops  suggested that we capitalize them to the user 

https://github.com/bitcointranscripts/transcription-review-front-end/pull/194#discussion_r1393826373

Comment (Extheoisah) [reply src/components/sideBarContentEdit/SelectField.tsx:225]: I have no preference but we should stick with what the issue states. besides, its common to use lowercase tags than uppercase e.g. github
image

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**